site/cds_rdm/webpack.py

```diff
@@ -19,6 +19,10 @@
                 "gltf_previewer_js": "./js/cds_rdm/previewers/gltf-previewer.js",
                 "gltf_previewer_css": "./less/cds_rdm/previewers/gltf-previewer.less",
             },
+            dependencies={
+                "three": "^0.180.0",
+                "three-addons": "^1.2.0",
```

Comment:
Not sure if there's any alternative but this was archived in 2021 and last publish 7 years ago. https://github.com/marcofugaro/three-addons.